src/Frontend-GNOME/Views/Chats/ChatView.cs

```diff
@@ -578,6 +585,25 @@ public virtual void Populate()
             _IsSynced = true;
         }
         
+        public virtual void UpdateLastSeenMessage()
+        {
+            _OutputMessageTextView.UpdateMarkerline();
+            
+            if (Frontend.EngineVersion >= new Version(0, 12)) {
```

Comment:
CS: can you use return here instead, this method has no other purpose from here than to push the change to the engine